README.md

```diff
@@ -53,6 +53,8 @@ If you're not familiar with Docker, it might be helpful to have a look at the Do
 * neqo: [Dockerfile](https://github.com/mozilla/neqo/blob/main/neqo-qns/Dockerfile) and [run_endpoint.sh](https://github.com/mozilla/neqo/blob/main/neqo-qns/run_endpoint.sh)
 * msquic: [Dockerfile](https://github.com/microsoft/msquic/blob/master/Dockerfile), [run_endpoint.sh](https://github.com/microsoft/msquic/blob/master/scripts/run_endpoint.sh) and [CI config](https://github.com/microsoft/msquic/blob/master/.azure/azure-pipelines.docker.yml)
 
+Note that the [online interop](https://interop.seemann.io/) runner requires `linux/amd64` architecture, so use `docker build` with `--platform linux/amd64` to create a compatible image if you build on a different architecture (e.g. "Apple silicon").
```

Comment:
Should we recommend a multi-platform build instead? It's nice to have images for both arm64 and amd64, e.g. https://hub.docker.com/repository/docker/martenseemann/quic-go-interop/tags?page=1&ordering=last_updated

Reply:
Agree. But we don't have to recommend it _instead_  I think ;-) ? We could just add that recommendation, the fact the online interop runner requires amd64 still holds....

Reply:
Sure, sounds good to me. Want to update the PR?

Reply:
done